client/src/components/BadReview/BadReviewControllers.tsx

```diff
@@ -38,6 +38,12 @@ export function BadReviewControllers({ courseTasks, courseId }: IBadReviewContro
     }
   }, [taskId, checkType, checkService, courseId]);
 
+  const downloadSolutionsUrls = async () => {
+    if (courseId && taskId) {
+      window.location.href = `/api/v2/courses/${courseId}/cross-checks/${taskId}/csv`;
```

Comment:
should it new tab?

Reply:
Not sure it's necessary, we do it similar way in other places where we use it, it causes immediate file download, for me it looks convenient